### PR TITLE
Improve legend example

### PR DESCRIPTION
**Pull request by davidmanzanares**
screenshot from 2018-11-07 11-27-26

### CODE REVIEW

File: examples/misc/legends/poly-legend-ma.html [hunk 1]
```diff
@@ -72,11 +72,17 @@ <h1>Percent Master's Degree</h1>
 
       colorLegend.data.forEach((legend) => {
         const colorHex = rgbToHex(legend.value);
-        const keyMin = legend.key[0].toFixed(2).replace('Infinity', '∞');
-        const keyMax = legend.key[1].toFixed(2).replace('Infinity', '+∞');
+        const keyMin = legend.key[0].toFixed(2);
```
Comment (VictorVelarde) [inline examples/misc/legends/poly-legend-ma.html]: 👍

**Comment by rochoa** [comment]
Superseded by #1133.

**Comment by makella** [comment]
oops the only thing is the `[` on one side and the `)` on the other.

**Comment by VictorVelarde** [comment]
That's open vs closed interval

**Comment by davidmanzanares** [comment]
That's a feature!

Not, really, it is a feature hahahaha: https://en.wikipedia.org/wiki/Interval_(mathematics)#Notations_for_intervals

I don't care to change it for a less math-like notation though.

**Comment by makella** [comment]
ohhhh haha... ok math wizards :)

for a choropleth legend, typically, you see `classmin - classmax` and rarely the more fancy notation we have. 

https://www.google.com/search?q=choropleth+map+legend&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiEnrLOwcLeAhVQT98KHcrQB5MQ_AUIEygB&biw=1854&bih=1160#imgrc=_

for simplicity's sake, is it ok if we make it more "traditional"?

thankssss

**Pull request closed without merging**